.github/workflows/job_keras3_backend.yml

```diff
@@ -0,0 +1,114 @@
+name: Keras 3 OpenVINO Backend
+
+on:
+  workflow_call:
+    inputs:
+      runner:
+        description: 'Machine on which the tests would run'
+        type: string
+        required: true
+      container:
+        description: 'JSON to be converted to the value of the "container" configuration for the job'
+        type: string
+        required: false
+        default: '{"image": null}'
+      affected-components:
+        description: 'Components that are affected by changes in the commit defined by the Smart CI Action'
+        type: string
+        required: true
+      python-version:
+        description: 'Python version to setup. E.g., "3.11"'
+        type: string
+        required: true
+
+permissions: read-all
+
+jobs:
+  Keras3_OpenVINO_Backend:
+    name: Keras 3 OpenVINO Backend
+    timeout-minutes: 10
+    defaults:
+      run:
+        shell: ${{ contains(inputs.runner, 'win') && 'pwsh' || 'bash' }}
+    runs-on: ${{ inputs.runner }}
+    container: ${{ fromJSON(inputs.container) }}
+    env:
+      INSTALL_DIR: ${{ github.workspace }}/install
+      INSTALL_WHEELS_DIR: ${{ github.workspace }}/install/wheels
+      KERAS_REPO: ${{ github.workspace }}/keras_repo
+      KERAS_HOME: ${{ github.workspace }}/keras_repo/.github/workflows/config/openvino
+
+    steps:
+      # Needed as ${{ github.workspace }} is not working correctly when using Docker
+      - name: Setup Variables
+        run: |
+          echo "INSTALL_DIR=$GITHUB_WORKSPACE/install" >> "$GITHUB_ENV"
+          echo "KERAS_REPO=$GITHUB_WORKSPACE/keras" >> "$GITHUB_ENV"
+          echo "KERAS_HOME=$GITHUB_WORKSPACE/keras/.github/workflows/config/openvino" >> "$GITHUB_ENV"
+
+      - name: checkout actions
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        timeout-minutes: 15
+        with:
+          sparse-checkout: |
+            .github/actions/setup_python
+            .github/actions/install_ov_wheels/action.yml
+
+      - name: Clone Keras 3 repository
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        timeout-minutes: 15
+        with:
+          repository: 'keras-team/keras'
+          path: ${{ env.KERAS_REPO }}
+          ref: 'master'
```

Comment:
Could it be pinned to a specific commit? Since this stage is included in the required ones, an update to the third-party repo could break this job. @mryzhov

Reply:
yes, it should be pinned. We intentionally anticipate that OpenVINO will still work for latest Keras master from which they release.